src/runtime-rs/crates/hypervisor/src/dragonball/mod.rs

```diff
@@ -201,41 +201,39 @@ impl Persist for Dragonball {
     }
 }
 
-impl From<NetworkConfig> for DragonballNetworkConfig {
-    fn from(value: NetworkConfig) -> Self {
-        let r = &value;
-        r.into()
-    }
-}
-
-impl From<&NetworkConfig> for DragonballNetworkConfig {
-    fn from(value: &NetworkConfig) -> Self {
-        let virtio_config = DragonballVirtioConfig {
-            iface_id: value.virt_iface_name.clone(),
-            host_dev_name: value.host_dev_name.clone(),
-            // TODO(justxuewei): rx_rate_limiter is not supported, see:
-            // https://github.com/kata-containers/kata-containers/issues/8327.
-            rx_rate_limiter: None,
-            // TODO(justxuewei): tx_rate_limiter is not supported, see:
-            // https://github.com/kata-containers/kata-containers/issues/8327.
-            tx_rate_limiter: None,
-            allow_duplicate_mac: value.allow_duplicate_mac,
-        };
-        let backend = match value.backend {
-            Backend::Virtio => DragonballBackend::Virtio(virtio_config),
-            Backend::Vhost => DragonballBackend::Vhost(virtio_config),
-        };
-
-        Self {
-            num_queues: Some(value.queue_num),
-            queue_size: Some(value.queue_size as u16),
-            backend,
-            guest_mac: value.guest_mac.clone().map(|mac| {
-                // We are safety since mac address is checked by endpoints.
-                DragonballMacAddr::from_bytes(&mac.0).unwrap()
-            }),
-            use_shared_irq: value.use_shared_irq,
-            use_generic_irq: value.use_generic_irq,
-        }
+/// Generate Dragonball network config according to hypervisor config and
+/// runtime network config.
+pub(crate) fn dragonball_network_config(
```

Comment:
The function name ` dragonball_network_config` is not clear. I think it'd better to change it to something that is more specific and easier to understand, such as "build_xxx", "setup_xxx", "init_xxx"...

Reply:
Fixed.